src/win7/win7-backend.cpp

```diff
@@ -288,7 +294,6 @@ namespace librealsense
                     DEVICE_NOTIFY_WINDOW_HANDLE);
                 if (data->hdevnotifyUVC == nullptr)
                 {
-                    UnregisterDeviceNotification(data->hdevnotifyHW);
```

Comment:
Why?

Reply:
This was an invalid call as it tried to do `UnregisterDeviceNotification(**nullptr**);`
which produces nothing.
Removing this call rectifies the flow